Update NetFlow documentation for GA (#18287)
DataDog/documentation: content/en/network_monitoring/devices/netflow.md

## Installation

To use NetFlow Monitoring with Network Device Monitoring, ensure you are using the [Agent][1] version 7.45 or newer.

> **Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is strongly recommended as this extra data can be used to enrich your flow records with information like the device name, model and vendor as well as the inbound/outbound interface name.

Reviewer: ```suggestion
**Note:** Configuring [metric collection from Network Device Monitoring][2] is not a requirement for sending NetFlow data, although it is strongly recommended as this extra data can be used to enrich your flow records with information such as the device name, model, and vendor, as well as the inbound/outbound interface name.
```

After saving your changes, [restart the Agent][4].

## Aggregation

> The Datadog Agent will automatically aggregate the received NetFlow data in order to limit the number of records sent to platform while maintaining most of the information. By default there is a 5 minutes aggregation interval, flow recording sharing the same identifying information (source and destination address and port, protocol etc.) will be aggregated together. Addtionnally the Datadog Agent can detect ephemeral port and remove them, this is why you may see Flows with `port:*`.

Reviewer: ```suggestion
The Datadog Agent automatically aggregates the received NetFlow data in order to limit the number of records sent to the platform while maintaining most of the information. By default there is a 5 minute aggregation interval, during which flow recordings which share the same identifying information (source and destination address and port, protocol, and so forth) will be aggregated together. Additionally, the Datadog Agent can detect ephemeral ports and remove them. As a result, you may see Flows with `port:*`.
```

## Enrichment

> Your NetFlow data is processed by the Datadog backend and they are enriched with the available metadata we have about your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.

Reviewer: ```suggestion
Your NetFlow data is processed by the Datadog backend and are enriched with the available metadata from your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.
```

> Your NetFlow data is processed by the Datadog backend and they are enriched with the available metadata we have about your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.

Reviewer 2: Should we explain or link to a page/section on how to configure namespace ?

> In case the NetFlow exporter IP is one of the device IPs but not the one configured on the SNMP integration, Datadog will try to find which device does this exporter IP belong to and will enrich your NetFlow data with is as long as the match is unique.

Reviewer: ```suggestion
If the NetFlow exporter IP is one of the device IPs, but not the one configured on the SNMP integration, Datadog will attempt to locate the device that the exporter IP belongs to, and will enrich your NetFlow data with it is as long as the match is unique.
```

## Visualization

> You can find the NetFlow page directly from the [Network Devices page][5].

Reviewer: ```suggestion
You can find the NetFlow page on the [Network Devices page][5].
```

{{< img src="network_device_monitoring/netflow/netflow_page.png" alt="NetFlow Page" >}}

> But your data is also available in Dashboards, Notebooks and more for more precise queries and for correlating with other sources of data.

Reviewer: ```suggestion
Your data is also available in Dashboards, Notebooks, and more for more precise queries and for correlating with other sources of data.
```

> But your data is also available in Dashboards, Notebooks and more for more precise queries and for correlating with other sources of data.

Reviewer 2: I think it can help to mention that the data is accessible via "NDM Traffic" data source.

> But your data is also available in Dashboards, Notebooks and more for more precise queries and for correlating with other sources of data.

Reviewer 2: Should we also mention that we have OOTB Dashboards for NetFlow and possible where there are or keyword to use to search those dashboards?
